joint_trajectory_controller/src/joint_trajectory_controller.cpp

```diff
@@ -1563,6 +1635,45 @@ void JointTrajectoryController::resize_joint_trajectory_point_command(
   }
 }
 
+bool JointTrajectoryController::set_scaling_factor(const double scaling_factor)
+{
+  if (scaling_factor < 0)
+  {
+    RCLCPP_WARN(
+      get_node()->get_logger(),
+      "Scaling factor has to be greater or equal to 0.0 - Ignoring input!");
+    return false;
+  }
+
+  RCLCPP_INFO(get_node()->get_logger(), "New scaling factor will be %f", scaling_factor);
```

Comment:
I think it should be possible to disable this logging message. Some one could have the idea of continuously streaming the scaling factor and this would lead to spamming the log/terminal

Reply:
We could set that output to an equality check with the current value and add a throttle to it. I think that might be a better approach than adding yet another config value.

Reply:
Just as a reminder @fmauch  ;)

Reply:
Thanks for the reminder, this should be resolved now.